lms/tests/test_flask_limiter.py

```diff
@@ -0,0 +1,25 @@
+from lms.tests import conftest
+from lms.lmsdb.models import User
+from lms.lmsweb import webapp
+
+
+class TestLimiter:
```

Comment:
Great changes.

We only need to add a test that regular login (with a single try) is working correctly.

Reply:
Also: add a test that when trying to `GET` the page `/login`, it doesn't apply the rate limit even after `LIMITS_PER_MINUTE` times.